app/ide-desktop/lib/client/watch.ts

```diff
@@ -94,7 +94,7 @@ const ALL_BUNDLES_READY = new Promise<Watches>((resolve, reject) => {
 
         console.log('Bundling content.')
         const contentOpts = contentBundler.bundlerOptionsFromEnv({
-            devMode: process.env.DEV_MODE !== 'false',
+            devMode: true,
```

Comment:
it should be fine; this is the development `watch` script after all. it might still be useful to allow overriding to test a prod build, not quite sure.

Reply:
on second thought, `esbuild` is no longer being used for dev mode so we are no longer relying on esbuild's auto reload functionality, so it should be safe to remove.